include/z64scene.h

```diff
@@ -74,87 +74,91 @@ typedef struct {
 typedef struct {
     /* 0x00 */ Gfx* opa;
     /* 0x04 */ Gfx* xlu;
-} MeshHeader01Entry; // size = 0x08
+} MeshHeaderDListsEntry; // size = 0x08
 
 typedef struct {
     /* 0x00 */ MeshHeaderBase base;
     /* 0x01 */ u8 numEntries;
-    /* 0x04 */ MeshHeader01Entry* entries;
-    /* 0x08 */ MeshHeader01Entry* entriesEnd;
-} MeshHeader0; // size = 0x0C
+    /* 0x04 */ MeshHeaderDListsEntry* entries;
+    /* 0x08 */ MeshHeaderDListsEntry* entriesEnd;
+} MeshHeaderAll; // size = 0x0C
 
 typedef enum {
-    /* 1 */ MESH_HEADER1_FORMAT_SINGLE = 1,
-    /* 2 */ MESH_HEADER1_FORMAT_MULTI
-} MeshHeader1Format;
+    /* 1 */ MESH_HEADER_PRERENDER_FORMAT_SINGLE = 1,
+    /* 2 */ MESH_HEADER_PRERENDER_FORMAT_MULTI
+} MeshHeaderPrerenderFormat;
 
 typedef struct {
     /* 0x00 */ MeshHeaderBase base;
-    /* 0x01 */ u8    format; // MeshHeader1Format
-    /* 0x04 */ MeshHeader01Entry* entry;
-} MeshHeader1Base; // size = 0x08
+    /* 0x01 */ u8    format; // MeshHeaderPrerenderFormat
+    /* 0x04 */ MeshHeaderDListsEntry* entry;
+} MeshHeaderPrerenderBase; // size = 0x08
 
 typedef struct {
-    /* 0x00 */ MeshHeader1Base base;
+    /* 0x00 */ MeshHeaderPrerenderBase base;
     /* 0x08 */ void* source;
     /* 0x0C */ u32   unk_0C;
     /* 0x10 */ void* tlut;
     /* 0x14 */ u16   width;
     /* 0x16 */ u16   height;
     /* 0x18 */ u8    fmt;
     /* 0x19 */ u8    siz;
-    /* 0x1A */ u16   mode0;
+    /* 0x1A */ u16   tlutMode;
     /* 0x1C */ u16   tlutCount;
-} MeshHeader1Single; // size = 0x20
+} MeshHeaderPrerenderSingle; // size = 0x20
 
 typedef struct {
     /* 0x00 */ u16   unk_00;
-    /* 0x02 */ u8    id;
+    /* 0x02 */ u8    index;
     /* 0x04 */ void* source;
     /* 0x08 */ u32   unk_0C;
     /* 0x0C */ void* tlut;
     /* 0x10 */ u16   width;
     /* 0x12 */ u16   height;
     /* 0x14 */ u8    fmt;
     /* 0x15 */ u8    siz;
-    /* 0x16 */ u16   mode0;
+    /* 0x16 */ u16   tlutMode;
     /* 0x18 */ u16   tlutCount;
 } BgImage; // size = 0x1C
 
 typedef struct {
-    /* 0x00 */ MeshHeader1Base base;
+    /* 0x00 */ MeshHeaderPrerenderBase base;
     /* 0x08 */ u8    count;
     /* 0x0C */ BgImage* list;
-} MeshHeader1Multi; // size = 0x10
+} MeshHeaderPrerenderMulti; // size = 0x10
 
 typedef struct {
     /* 0x00 */ Vec3s pos;
-    /* 0x06 */ s16 unk_06;
+    /* 0x06 */ s16 radius;
```

Comment:
Could this match as a `Sphere16` substruct?

Reply:
It does, good idea. btw I was still thinking in 2D and was still seeing this as a cylinder, thanks for making sphere remarks :)

Reply:
Well actually yes it matches, but it won't work until ZAPD is updated